setup

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+source environment.conf
+
+if [ ! -d $promethee_path ]; then
+	git clone $promethee_url -b $promethee_branch
+	git --git-dir=$promethee_path/.git checkout --detach $promethee_version
+fi
+
+make -C $promethee_path
+
+if [ ! -d $tiff_utils_path ]; then
+	git clone $tiff_utils_url
+	git --git-dir=$tiff_utils_path/.git checkout --detach $tiff_utils_version
+fi
+
```

Comment:
@Matheus7OP it would be better to have a Makefile instead of compile stuff in the coordination script